.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
create .gitignore file in the root 
add .idea folder to it
such files should never appear in repo